App.js

```diff
@@ -13,6 +12,7 @@ import SidebarCollapsibleWrapper from "./app/SidebarCollapsibleWrapper";
 // Required to make Material UI work with touch screens.
 import injectTapEventPlugin from "react-tap-event-plugin";
 import Checkbox from "./composites/Plugin/Shared/components/Checkbox";
+import SnippetEditor from "./app/SnippetEditorWrapper";
```

Comment:
`Wrapper` naming does not seem to be explanatory, have you considered using `Example`?